node/services/config.go

```diff
@@ -1 +1,44 @@
 package services
+
+import (
+	"encoding/hex"
+
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/multiformats/go-multiaddr"
+	tmbytes "github.com/tendermint/tendermint/libs/bytes"
+)
+
+type Config struct {
+	// GenesisHash is the Block/Header hash that Nodes use as starting point for header synchronization.
+	// Only affects the node once on initial sync.
+	GenesisHash string
```

Comment:
Q: Can this be any trusted header hash that the user chooses or does it have to be from genesis? Because the var name makes it sound like this is the genesis hash but the comment just speaks of it as a subjective starting point?

Reply:
It's like something in between. It can only be used one time as genesis, but it can go from any height and not specifically 0. Not sure how to name it.

Reply:
`TrustedHeaderHash`?

Reply:
Second `TrustedHeaderHash`.

Reply:
Changing this to TrustedHeaderHash